catalogue/aarch64/tests/LB+rel+CAS-no-RtM-data.litmus

```diff
@@ -0,0 +1,19 @@
+AArch64 LB+rel+CAS-no-RtM-data
+
+{
+  x = 0;
+  y = 0;
+
+  0: X0 = x;      1: X0 = x;
+  0: X1 = y;      1: X1 = y;
+}
+
+P0            | P1                ;
+
+LDR W3, [X0]  | LDR W3, [X1]      ;
+MOV W4, #1    | EOR W4, W3, W3    ;
+STLR W4, [X1] | ADD W4, W4, #1    ;
+              | MOV W5, #2        ;
+              | CAS W5, W4, [X0]  ;
```

Comment:
In this case (CAS-no) the write of CAS can only set x to 0. Is the post-condition correct?